### PR TITLE
[s2t] DataParallel init method changed, fixed conformer could not multi-gpu training and don't affect dy2st

### PR DESCRIPTION
**Pull request by Zth9730**
### PR types

 Bug fixes
### PR changes

Models
### Describe

fix dp init

### CODE REVIEW

File: paddlespeech/s2t/training/trainer.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 import paddle
 from paddle import distributed as dist
+dist.init_parallel_env()
```
Comment (yt605155624) [inline paddlespeech/s2t/training/trainer.py]: world_size = paddle.distributed.get_world_size()
    if world_size > 1:
        paddle.distributed.init_parallel_env()

File: paddlespeech/s2t/training/trainer.py [hunk 2]
```diff
@@ -19,6 +19,10 @@
 
 import paddle
 from paddle import distributed as dist
+world_size = dist.get_world_size()
```
Comment (zh794390558) [inline paddlespeech/s2t/training/trainer.py:22]: 加上注释说明下原因。

**Review by zh794390558 — COMMENTED** [review]
代码有format吗？

**Review by zh794390558 — COMMENTED** [review]
Title说明下修复的bug是什么

File: paddlespeech/s2t/training/trainer.py (hunk 2 above)
Comment (Zth9730) [reply paddlespeech/s2t/training/trainer.py:22]: format过

**Review by zh794390558 — APPROVED** [review]
LGTM

**Pull request merged**